draft-ietf-quic-ack-frequency.md

```diff
@@ -167,7 +167,7 @@ However, as discussed in {{implementation}}, a unilateral reduction in
 acknowledgement frequency can lead to undesirable consequences for congestion
 control and loss recovery. {{QUIC-TRANSPORT}} specifies a simple delayed
 acknowledgment mechanism ({{Section 13.2.1 of QUIC-TRANSPORT}}) without any
-ability for the data sender to impact this behavior.  A data sender's constraints
+ability for the data sender to control this behavior.  A data sender's constraints
```

Comment:
```suggestion
ability for the data sender to influence this behavior.  A data sender's constraints
```

We use influence above, and I think it's slightly better than control in this case.

Reply:
If /influence/ is more consistent that's fine with me, the word I wanted to avoid was /impact/.